ibutex.py

```diff
@@ -10,17 +10,21 @@
 parser.add_argument('-q', '--quick', action="store_true", dest='quick', default=False, help='quick build, single run')
 parser.add_argument('-i', '--include', type=str, metavar='file/dir', nargs='+', help='files or folder that need to be linked to the builddir')
 parser.add_argument('--build-only', action="store_false", dest='showpdf', default=True, help='show the compiled documend afterwards')
+parser.add_argument('--biber', action="store_true", dest='biber', default=False, help='use biber instead of bibtex')
```

Comment:
We will not go down this rabbithole of madness. Move line 6 (`bibtexbase`) into a commandline flag `-b/--bibcmd` and let it default to `bibtex`. I intend to do similar things for a `-l/--latexcmd` to get rid of `latexbase` in line 5. We will not start to add cli-flags for single programs, madness of unmaintainable code lies there....

(and make sure you commit with `--amend` to ensure that we stick with two commits here)